main/params.go

```diff
@@ -120,6 +122,8 @@ func init() {
 
 	// IP:
 	consensusIP := fs.String("public-ip", "", "Public IP of this node")
+	fs.BoolVar(&Config.DynamicConsensusIP, "dynamic-public-ip", false, "If true, resolve ip from ifconfig.co")
+	fs.DurationVar(&Config.DynamicConsensusUpdate, "dynamic-public-ip-update", 1*time.Minute, "Dynamic IP update time")
```

Comment:
ifconfig.co says that it rate limits you to one request per minute, so we should make this larger to ensure we don't get rate limited (90 seconds or 2 minutes, perhaps?)

Reply:
i think it's ok to do 1 minute.

Reply:
I am going to use this for the NAT update and Dynamic IP update..  So I default to 5min.  It can be configured by user now.